### DISCUSSION TITLE
wide ?

### DISCUSSION THREAD

**Discussion by ithakaa** [usememos/memos#3576] [General]
Is there a way to edit the width of the editable page? why so narrow ?

**Comment by rdiezsj** [comment]
Same here. Is annoying

**Comment by MehadND** [comment]
I think you can try to do it with custom css in settings.

**Comment by rdiezsj** [comment]
I don’t know how to work with CSS, and there is not documentation for that. Do you have any example?

**Comment by RoccoSmit** [comment]
Try going to `settings` -> `system` -> `additional style` and overriding the style that stops the column from going wider

```
section.max-w-5xl {
  max-width: 100%;
}
```
note, this will override the current max width from 1024px to the full size of your screen. You can modify the value to get the % (or px) you prefer.

**Reply by rdiezsj** [reply]
Thanks for the help, but that's not exactly what you are looking for. The indicated code sets the timeline display width to 100%, but not the editor display width when editing a note.
How do I know which style I need to modify?

![image](https://github.com/user-attachments/assets/9306d7c4-9a7b-4e4a-93c7-3bf7f012266b)

**Reply by VaultBoy88** [reply]
> Try going to `settings` -> `system` -> `additional style` and overriding the style that stops the column from going wider
> 
> ```
> section.max-w-5xl {
>   max-width: 100%;
> }
> ```
> 
> note, this will override the current max width from 1024px to the full size of your screen. You can modify the value to get the % (or px) you prefer.

Folks, this does not work for me. The main window with all the memos is very narrow on 4k screen. 

<img width="1892" height="673" alt="image" src="https://github.com/user-attachments/assets/9a804987-f5cb-40d9-8769-e38d8d034c9a" />

Added the stanza to additional style but no effect. Can anyone help me figure this out please?

<img width="499" height="340" alt="image" src="https://github.com/user-attachments/assets/0b308879-a20f-4910-bcbd-7a76ed5a1ad2" />

**Reply by VaultBoy88** [reply]
Kind of answered the question with gemini. This works on 0.25.3.

`.max-w-2xl, 
.max-w-5xl, 
#root main section div.pl-72 > div {
  max-width: calc(100vw - 20rem) !important;
  width: 100% !important;
  margin: 0 auto !important;
}`

Is this optimal or is there a better/right way?

**Comment by MehadND** [comment]
I think you should try this class name `memo-editor-dialog`. If that does not work then probably join the discord server and ask there.

**Comment by RoccoSmit** [comment]
Try

```
.memo-editor-dialog .dialog-container {
    width: 100%;
}

.memo-editor-dialog .dialog-container .w-\[40rem\] {
    width: 100%;
}
```
The `.dialog-container` change sets the % of the screen you want to use and the `40rem` change sets the text inside the dialog to use the full width of the dialog

**Reply by rdiezsj** [reply]
Wow, this works!!! thank you very much
